plugins/formal-verification/formver.viper/src/org/jetbrains/kotlin/formver/viper/errors/ConsistencyError.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver.viper.errors
+
+import org.jetbrains.kotlin.formver.viper.ast.Position
+
+sealed interface ConsistencyError : VerifierError {
```

Comment:
Why is it necessary to split out this interface?

Reply:
So we can categorize specific types of errors more precisely since the errors generated by the verifier can be:
1. of consistency: the generated code is not valid, so we report a debug message for ourselves
2. of verification: the verification process failed, so we try to understand what went wrong and generate a proper message